content/docs/overview.md

```diff
@@ -617,7 +617,7 @@ multiply :: proc(x, y: int) -> int {
 fmt.println(multiply(137, 432))
 ```
 
-Continuing the C family tradition, everything in Odin is passed by value (rather than by reference, e.g. FORTRAN, Java, etc). However, Odin differs from the C/C++ tradition in that all procedure parameters in Odin are immutable values. This allows for numerous optimizations with the Odin calling conventions (`"odin"` and `"contextless"`) which would not be possible with the original C tradition of always passing a copy of the thing that has been passed.
+Continuing the C family tradition, everything in Odin is passed by value (rather than by reference, e.g. FORTRAN, Java, etc). However, Odin differs from the C/C++ tradition in that all procedure parameters in Odin are immutable values. This allows for numerous optimizations with the Odin calling conventions (`"odin"` and `"contextless"` explained [here](#calling-conventions)) which would not be possible with the original C tradition of always passing a copy of the thing that has been passed.
```

Comment:
I would change it so it's "calling conventions" that's linked to `#calling-conventions`, e.g.:

This allows for numerous optimizations with the Odin [calling conventions](#calling-conventions) (`"odin"` and `"contextless"`, which would not be possible with the original C tradition of always passing a copy of the thing that has been passed.

Reply:
nice that reads better :+1: